sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -92,9 +93,9 @@ public NettyAsyncHttpClientBuilder(HttpClient nettyHttpClient) {
         if (this.baseHttpClient != null) {
             nettyHttpClient = baseHttpClient;
         } else if (this.connectionProvider != null) {
-            nettyHttpClient = HttpClient.create(this.connectionProvider);
+            nettyHttpClient = HttpClient.create(this.connectionProvider).resolver(DefaultAddressResolverGroup.INSTANCE);
```

Comment:
Is this not supported in OkHttp?

Reply:
It is possible to configure the DNS resolver in OkHttp as well but it doesn't run into the same issue being seen with Reactor Netty.

Reply:
Since this is applicable to all HTTP clients, is this something we should put on `HttpClientOptions` so that the user doesn't have to go build their own HTTP client when they have to configure this?

Reply:
Issue is that the types being used aren't in the base Java library so there will need to be a wrapper interface